packages/tools/nodeEditor/src/graphEditor.tsx

```diff
@@ -247,19 +247,19 @@ export class GraphEditor extends React.Component<IGraphEditorProps, IGraphEditor
             return this._graphCanvas.findNodeFromData(block);
         };
 
-        this.props.globalState.hostDocument.addEventListener("keydown", (evt) => {
+        this.props.globalState.hostDocument.addEventListener("keydown", async (evt) => {
             if (this._historyStack.processKeyEvent(evt)) {
                 return;
             }
 
-            this._graphCanvas.handleKeyDown(
+            await this._graphCanvas.handleKeyDownAsync(

```

Comment:
i'd prefer to use

`void this._graphCanvas.handleKeyDownAsync(` instead of using async/await if you do not mind? Because here we are using it that way